src/main/java/com/github/hcsp/Home.java

```diff
@@ -16,5 +16,9 @@ public static void createTwoCats() {
         // Create two new Cats named "Black" and "White"
         // then assign them to "black" and "white" static variable respectively
         // 在这里创建两个Cat实例，分别赋值给"black"和"white"静态变量，然后将它们的"name"成员变量分别设置为"Black"和"White"
+        Cat black = new Cat();
+        Cat white = new Cat();
```

Comment:
```suggestion
        white = new Cat();
```

要求是：分别赋值给"black"和"white"静态变量，而你赋值给了两个局部变量，请体会一下区别。